src/mspdb.cpp

```diff
@@ -51,7 +51,11 @@ bool getInstallDir(const char* version, char* installDir, DWORD size)
 	strcat(key, version);
 
 	HKEY hkey;
-	if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, key, 0, KEY_QUERY_VALUE, &hkey) != ERROR_SUCCESS)
+	if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, key, 0, KEY_QUERY_VALUE
+#if _M_X64
+		| KEY_WOW64_32KEY
+#endif

```

Comment:
I don't like slicing an expression with preprocessor directives these days. Instead recommend defining some `KEY_OPEN_FLAGS` somewhere before that includes `KEY_WOW64_32KEY` for x64 builds.